boards/arm/mimxrt685_evk/flash_flexspi_nor_lut.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "flash_mcux_flexspi_nor_lut_index.h"
+
+static const uint32_t flash_flexspi_nor_lut[][4] = {
```

Comment:
Can this LUT be used for other octal flashes or is it specific to MX25UM51345G?

Reply:
It is specific to MX25UM51345G. JEDEC standard is not valid for octal mode.

Reply:
Not just memory specific but to do XIP or to trigger a read command sequence from LUT, dummy cycles are required. And dummy cycles requires as per memory model's data sheet varies as per frequency at which flash is operated. So along with memory model LUT is also platform/board specific.